bidirectional.py

```diff
@@ -0,0 +1,111 @@
+import collections
+from graph import Graph
+
+Oradea="Oradea"
+Zerind="Zerind" 
+Arad="Arad" 
+Timisoara="Timisoara"
+Lugoj="Lugoj"
+Mehadia="Mehadia" 
+Drobeta="Drobeta"  
+Craiova="Craiova"   
+Sibiu="Sibiu"
+Rimnicu_Vilcea="Rimnicu Vilcea"   
+Fagaras="Fagaras"
+Pitesti="Pitesti"
+Giurgiu="Giurgiu"
+Bucharest="Bucharest"    
+Urziceni="Urziceni"   
+Eforie="Eforie"  
+Hirsova="Hirsova"   
+Vaslui="Vaslui"  
+Iasi="Iasi"  
+Neamt="Neamt"   
+
+cities = {}
```

Comment:
I like how you read the coordinate data from the text file. But I've lately written a Heuristic class in the 'heuristic.py' file that does the job. It can be used across the repo, so we're using that.